pkgs/development/python-modules/boddle/default.nix

```
@@ -0,0 +1,34 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pythonOlder
+, bottle
+}:
+
+buildPythonPackage rec {
+  pname = "boddle";
+  version = "0.2.9";
+  disabled = pythonOlder "3.7";
+
+  src = fetchFromGitHub {
+    owner = "keredson";
+    repo = pname;
+
+    # upstream doesn't tag releases
+    rev = "74413e4c5b39a37ea1694fab07b459ccd2998390";
+    sha256 = "sha256-iyxgdUM3NEbDaRd8ogCsxyNPF0nMuIV2S4FiFAZJXrY=";
+  };
+
+  propagatedBuildInputs = [ bottle ];
+
+  pythonImportsCheck = [
+    "boddle"
+  ];
+
+  meta = with lib; {
+    description = "Unit testing tool for Python's bottle library";
+    homepage = "https://github.com/keredson/boddle";
+    license = licenses.lgpl21;
```

The `setup.py` says [LGPLv3](https://github.com/keredson/boddle/blob/master/setup.py#L27) and the LICENSE file [LGPLv2.1](https://github.com/keredson/boddle/blob/master/LICENSE).

```suggestion
    license = licenses.lgpl21Only;
```

https://github.com/keredson/boddle/issues/15